pkg/ingestor/parser/slsa/parser_slsa.go

```diff
@@ -19,16 +19,20 @@ import (
 	"context"
 	"errors"
 	"fmt"
+
 	"strings"
+	"time"
 
 	jsoniter "github.com/json-iterator/go"
 
-	"github.com/in-toto/in-toto-golang/in_toto"
+	slsa1 "github.com/in-toto/attestation/go/predicates/provenance/v1"
+	attestationv1 "github.com/in-toto/attestation/go/v1"
 	scommon "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/common"
 	slsa01 "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v0.1"
 	slsa02 "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v0.2"
-	slsa1 "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v1"
+	smtslsa1 "github.com/in-toto/in-toto-golang/in_toto/slsa_provenance/v1"
```

Comment:
I dont think we should have this anymore?

Reply:
Actually we need `"https://slsa.dev/provenance/v1"` to check the slsa version and it is defined in in-toto-golang [here](https://github.com/in-toto/in-toto-golang/blob/72262c0fd3a9c392440040c04336174eddac427b/in_toto/slsa_provenance/v1/provenance.go#L11). Since slsa1 was used for `"github.com/in-toto/attestation/go/predicates/provenance/v1"`, I used smtslsa1, implying statement for slsa 1. Should I change the name or hardcode the value in parser?